Makefile

```diff
@@ -77,8 +78,14 @@ tarball: $(PROMU)
 	@$(PROMU) tarball --prefix $(PREFIX) $(BIN_DIR)
 
 docker:
+ifeq ($(MACH), ppc64le)
+	@echo ">> building docker image for ppc64le"
+	$(eval FILE_SUFFIX=.ppc64le)
```

Comment:
I'd call it `DOCKER_FILE` and pass the full file. Could also happen setting as variable at the top.